json/gitee_issues_timing.json

```diff
@@ -43,7 +43,7 @@
                     "grimoire_creation_date",
                     "desc"
                 ],
-                "title": "Search:gittee_pull_request:false",
+                "title": "Search:gitee_pull_request:false",
```

Comment:
Thank you @wengzhenjie . The id also need to be corrected. 

https://github.com/X-lab2017/grimoirelab-sigils/blob/05125b2f23cd9ddb4d726ed82e94599df356274d/json/gitee_issues_timing.json#L32